core/include/drivers/firewall/firewall.h

```diff
@@ -0,0 +1,113 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2024, STMicroelectronics
+ */
+
+#ifndef __DRIVERS_FIREWALL_H
+#define __DRIVERS_FIREWALL_H
+
+#include <compiler.h>
+#include <drivers/firewall/firewall_device.h>
+#include <stddef.h>
+#include <tee_api_defines.h>
+#include <types_ext.h>
+
+struct firewall_controller_ops;
```

Comment:
Not used. I think this ops struct could be defined, and referred from `struct firewall_controller`. I would allows driver to use a `static const struct firewall_controller_ops` reference.